charts/unpoller/README.md

```diff
@@ -0,0 +1,54 @@
+# unpoller
+
+![Version: 2.11.2](https://img.shields.io/badge/Version-2.11.2-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v2.11.2](https://img.shields.io/badge/AppVersion-v2.11.2-informational?style=flat-square)
+
+A Helm chart for unpoller, a unifi prometheus exporter
+
+**Homepage:** <https://github.com/unpoller/unpoller>
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| XXXX? | <XXXX?> |  |
```

Comment:
Right?

Reply:
Yes, it is autogenerated with whatever is on Chart.yaml, once we decide what goes in there it will be re-generated.